tokio/src/stream/mod.rs

```diff
@@ -0,0 +1,93 @@
+//! Stream utilities for Tokio.
+//!
+//! `Stream`s are an asynchoronous version of standard library's Iterator.
```

Comment:
typo:
```suggestion
//! `Stream`s are an asynchronous version of the standard library's Iterator.
```

Reply:
Also, I think I'd prefer it if we defined streams from first principles and _then_ compared them to `Iterator`. IMO it's a little weird to define anything _just_ as "like `std::whatever`, but async" — there might even be new Rust users who aren't familiar with std yet.

How about something like:
> A `Stream` is an asynchronous sequence of values. It can be thought of as an asynchronous version of the standard library's `Iterator` trait.

Reply:
(finally, I think we should link to `Iterator` here)